recipes/nutch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nutch" %}
+{% set version = "1.10.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "17cf195db88b11c5d283448fe362d483e78db58dc564339583de629790103782" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    - nutch-python = nutch.nutch:main
+
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+
+  run:
+    - python
+    - setuptools
+    - requests
```

Comment:
Are we just assuming that the user has their own version of nutch? The conda recipe in the upstream repo makes it a requirement:

https://github.com/chrismattmann/nutch-python/blob/master/conda.recipe/meta.yaml#L45